src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -37,19 +39,125 @@
  * @author Dragan Bozanovic (bozanovicdr@gmail.com)
  * @version $Id$
  * @since 0.25
- * @todo #377:30min/DEV We need more tests for TkProxy.
+ *
  *  The tests should verify the different HTTP methods (GET, POST, etc),
  *  as well as the different combinations of request/response headers.
  */
 public final class TkProxyTest {
 
     /**
+     * Test string.
+     */
+    public static final String HELLO_WORLD = "hello, world!";
+
+    /**
+     * Format string.
+     */
+    public static final String FORMAT = "%s:%d";
+
+    /**
+     * Dash constant.
+     */
+    public static final String DASH = "/";
+
+    /**
+     * Response string.
+     */
+    public static final String HEADER = "X-Takes-TkProxy:";
+
+    /**
+     * OK response.
+     */
+    public static final String OK = "HTTP/1.1 200 OK";
```

Comment:
@aschworer should be inlined as well
Try to avoid creating such strings as much ass possible. They are just a noise.